### PR TITLE
Docs updated, Prerequisites section [20.1]

### PR DESCRIPTION
**Pull request by Karm**
SSIA
Tested also on on "clean-room" Ubuntu.

### CODE REVIEW

**Comment by Karm** [comment]
@zakkak Updated, ready to be merged.

**Review by zakkak — APPROVED** [review]

File: README-Mandrel.md [hunk 1]
```diff
@@ -39,12 +39,14 @@ Mandrel's `native-image` depends on the following packages:
 
 On Fedora/CentOS/RHEL they can be installed with:
 ```bash
-dnf install glibc-devel zlib-devel gcc
+dnf install glibc-devel zlib-devel gcc libstdc++-static
```
Comment (zakkak) [inline README-Mandrel.md:42]: Missing `feetype-devel`?

Comment (zakkak) [reply README-Mandrel.md:42]: Also, shouldn't we include `libstdc++-static` in the list above?

**Comment by zakkak** [comment]
Closing as 20.1 is no longer supported

**Pull request closed without merging**